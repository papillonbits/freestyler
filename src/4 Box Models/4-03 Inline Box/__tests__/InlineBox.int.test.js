import { regular } from './InlineBox.int.story'

describe('<InlineBox />', () => {
  describe('Render', () => {
    test('must match regular()', () => {
      expect(global.renderToJSON(regular())).toMatchSnapshot()
    })
  })
})
