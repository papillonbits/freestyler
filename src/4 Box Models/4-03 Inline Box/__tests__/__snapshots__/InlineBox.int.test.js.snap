// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<InlineBox /> Render must match regular() 1`] = `
<div>
  <div
    class="inline-box"
  >
    <h1>
      Inline Box
    </h1>
    <div
      class="container"
    >
      <span
        class="default"
      >
        BEFORE
      </span>
      <span
        class="box"
      >
        ← Left   ↑ Top   Bottom ↓   Right → 
      </span>
      <span
        class="default"
      >
        AFTER
      </span>
    </div>
  </div>
</div>
`;
